src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+type Props = {
+  todos: Todo[];
+  tempTodo: Todo | null;
+  deleteTodo: (id: number) => void;
+  isProcessing: Todo | null;
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  tempTodo,
+  deleteTodo,
+  isProcessing,
+}) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {todos.map((todo) => (
+        <TodoItem
+          key={todo.id}
+          todo={todo}
+          deleteTodo={deleteTodo}
+          isProcessing={isProcessing}
+        />
+      ))}
+
+      {!!tempTodo && (
+        <TodoItem
+          key={tempTodo.id}
+          todo={tempTodo}
+          deleteTodo={() => {}}
```

Comment:
You can make it an optional props, then you won't need to pass an empty function